src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,77 @@
+/* eslint-disable no-console */
+/* eslint-disable react/prefer-stateless-function */
 import React from 'react';
 import './CurrentUser.scss';
+import { User } from '../../type/user';
+import { getUserById } from '../../api';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  userId: number;
+  handleClearUser: () => void;
+}
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+interface State {
+  user: User;
+}
+
+export class CurrentUser extends React.Component<Props, State> {
+  state: State = {
+    user: {
+      name: '',
+      id: 0,
+      email: '',
+      phone: '',
```

Comment:
just save all these variables inside state, no need for user prop

Reply:
What a benefit doing so?

Reply:
> just save all these variables inside state, no need for user prop

Done